t/cobrand/tfl.t

```diff
@@ -864,6 +875,21 @@ for my $test (
             'Trees'
         ],
     },
+    {
+        host => 'hackney.fixmystreet.com',
+        name => "test no hackney categories on red route",
+        lat => 51.552287,
+        lon => -0.063326,
+        expected => [
+            'Bus stops',
+            'Flooding',
+            'Grit bins',
+            'Pothole',
+            'Timings',
+            'Traffic lights',
+            'Trees'
+        ],
+    },
```

Comment:
Do we want also to test the message we don't want to be shown isn't being shown? 🤷